src/javascript/ast-value.ts

```diff
@@ -28,10 +28,30 @@ function literalToValue(literal: babel.Literal): LiteralValue {
   if (babel.isNullLiteral(literal)) {
     return null;
   }
+  if (babel.isTemplateLiteral(literal)) {
+    return templateLiteralToValue(literal);
+  }
   // Any other literal value is treated as undefined.
   return undefined;
 }
 
+function templateLiteralToValue(literal: babel.TemplateLiteral): LiteralValue {
+  const len = literal.quasis.length - 1;
+  let value: LiteralValue = '';
+
+  for (let i = 0; i < len; i++) {
+    value += literal.quasis[i].value.raw;
+    const v = expressionToValue(literal.expressions[i]);
+    if (v !== undefined) {
```

Comment:
if `v === undefined` then this function must return `undefined`, as we can't give a confident evaluation of the template literal

... should really rewrite this family of functions to be a Result<LiteralValue>